find-datalad-repos.py

```diff
@@ -400,6 +401,16 @@ def get_datalad_repos(self) -> List[DataladRepo]:
         results.sort(key=attrgetter("name"))
         return results
 
+    def repo_exists(self, repo_fullname: str) -> bool:
+        r = self.session.get(f"{self.API_URL}/repos/{repo_fullname}")
+        if r.ok:
+            return True
+        elif r.status_code == 404:
+            return False
+        else:
+            r.raise_for_status()
+            raise AssertionError("Unreachable")
```

Comment:
we wouldn't want the script to just crash if someone e.g. makes them private or whatnot -- should we just in this case announce them as `Gone`?

Reply:
I think private repos return 404, but I've added mapped 403 to Gone anyway.